python-backend/tests/mines/permit/resources/test_permit_amendment_resource.py

```diff
@@ -32,7 +32,7 @@ def test_get_permit_amendment_not_found(test_client, db_session, auth_headers):
 
 
 # #POST
-@pytest.mark.skip(
+@pytest.mark.xfail(
```

Comment:
this means we run these tests explicitly using `pytest -m xfail --runxfail tests/`
`-m xfail` means on collect the tests with that mark
`--runxfail` means runs xfail tests normally.